exercises/concept/meltdown-mitigation/conditionals.py

```diff
@@ -1,8 +1,11 @@
+""" Meltdown Mitigation exercise """
+
+
 def is_criticality_balanced(temperature, neutrons_emitted):
     """Verify criticality is balanced.
 
-    :param temperature: int
```

Comment:
Format:  `:param <name>: <type> [- optional text]`

Don't lose the type here.

Reply:
Should we stay with `int` or generalize to `float` ?

Reply:
It doesn't matter much. I think `int` is cleaner and simpler and that'd be my preference. Plus, if it isn't broken, don't fix it. `int` works just fine.

If you want to get fancy, you can go with [numbers.Real](https://docs.python.org/3/library/numbers.html#numbers.Real) :)

Reply:
Let's stay with `int` then, also tests feed ints. It's just conceptually strange for me to have quantities like temperature, voltage, current,... integers.

Reply:
Given that digital sensors produce discrete levels, `int` makes a lot of sense. Getting a float from a sensor involves converting that discrete level.